client/src/components/CategoriesScreen/CategoriesScreen.tsx

```diff
@@ -77,26 +89,52 @@ function CategoriesScreen() {
                       setModalEditVisible(true);
                     }}
                   >
-                    <Text style={styles.buttonEdit}>{LOCALES.EDIT}</Text>
+                    <FormattedMessage
+                      id="EDIT"
+                      defaultMessage={LOCALES_EN.EDIT}
+                    >
+                      {(msg) => <Text style={styles.buttonEdit}>{msg}</Text>}
+                    </FormattedMessage>
                   </Pressable>
                   <Pressable
                     onPress={() => {
                       setSelectedCategory(category);
                       handleDeleteCategory(category);
                     }}
                   >
-                    <Text style={styles.buttonEdit}>{LOCALES.DELETE}</Text>
+                    <FormattedMessage
+                      id="DELETE"
+                      defaultMessage={LOCALES_EN.DELETE}
+                    >
+                      {(msg) => <Text style={styles.buttonEdit}>{msg}</Text>}
+                    </FormattedMessage>
                   </Pressable>
                 </View>
               </View>
             ))}
           </View>
+          <View style={styles.categories}>
+            <FormattedMessage
+              id="STATISTICS"
+              defaultMessage={LOCALES_EN.STATISTICS}
+            >
+              {(msg) => <Text style={styles.title}>{msg}</Text>}
+            </FormattedMessage>
+            <View style={styles.chart}>
+              <BarChart data={getChartData(allCategories)} />
+            </View>
+          </View>
         </View>
         <Pressable
           style={styles.button}
           onPress={() => setModalAddVisible(true)}
         >
-          <Text style={styles.buttonText}>{LOCALES.ADD_NEW_CATEGORY}</Text>
+          <FormattedMessage
```

Comment:
This component has a lot of duplication, it would be nice to have a separate component and just pass properties